### PR TITLE
Fix preformatted regex

### PR DESCRIPTION
**Pull request by thockin**
@erictune @lavalamp

### CODE REVIEW

**Comment by lavalamp** [comment]
Hm... I'll try to look tomorrow.

File: cmd/mungedocs/util.go [hunk 1]
```diff
@@ -141,6 +134,13 @@ type fileBlock struct {
 
 type fileBlocks []fileBlock
 
+var (
+	// Finds all preformatted block start/stops.
+	preformatRE    = regexp.MustCompile("^\\s*```")
+	notPreformatRE = regexp.MustCompile("^\\s*```.*```")
+	preformatEndRE = regexp.MustCompile("^\\s*```")
```
Comment (lavalamp) [inline cmd/mungedocs/util.go]: Might as well just reuse preformatRE if they are the same. I had it separate to parse some (incorrect?) md files we had, but they've been fixed, I guess.

Comment (thockin) [inline cmd/mungedocs/util.go]: We could distinguish that begin is ^```\s*.*\s*$ while end is ^```\s*$ ?

**Comment by thockin** [comment]
You were right.  Same regex now.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**